langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -51,6 +51,19 @@ public type Timestamp readonly & object {
 
 # Returns information about the current transaction.
 #
+# ```
+# function createEntity() {
+#     transaction {
+#         transaction:Info info = transaction:info();
```

Comment:
I have a few questions related to this PR.
1. Do we need to show the outputs? 
    Example output for `transaction:info();` (long output)
    ```
    {"xid": [100,102,53,51,97,57,57,51,45,101,48,102,101,45,52,99,100,48,45,97,56,57,101,45,49,98,50,49,99,49,102,49,50,49,102,100],"retryNumber":0,"prevAttempt":null,"startTime":2022-12-20 10:53:25,802}
    ```
2. Do we need to show a simpler code example?
    Example:
    show only `transaction:Info info = transaction:info();` instead of the below code snippet
    ```ballerina
    transactional function df() returns error? {
         transaction:Info info = transaction:info();
    }
    ````
    We need to show 2 different contextual use cases for some of the lang-lib functions.

@mohanvive @MaryamZi @pcnfernando any idea?

Reply:
For 1, let's print the output, but shorten the xid. IMO, it doesn't need to be the actual byte[] since its generated randomly.
For 2, +1 for existing sample